react/features/shared-video/components/native/AbstractVideoManager.ts

```diff
@@ -208,34 +208,57 @@ class AbstractVideoManager extends PureComponent<IProps> {
     /**
      * Seeks video to provided time.
      *
-     * @param {number} time
+     * @param {number} _time - Time to seek to.
+     * @returns {void}
      */
-    seek: (time: number) => void;
+    seek(_time: number) {
```

Comment:
why add implementations for the formerly abstract methods?

Reply:
My bad. I fixed the problem the JS way (giving the methods an implementation) instead of the TS way (converting to abstract). Updated to use TS `abstract` now